### DISCUSSION TITLE
Is vaultwarden fully legal because it is a copy of Bitwarden

### DISCUSSION THREAD

**Discussion by lukasj98** [dani-garcia/vaultwarden#4220] [Q&A]
Is vaultwarden fully legal because it is a copy of Bitwarden and it has paid features for free 
Why is this legal because with bitwarden you have to pay

**Comment by eseelke** [comment]
Checkout https://bitwarden.com/open-source/.

**Reply by lukasj98** [reply]
Open Source doesn't mean that it is allowed to copy it...

**Reply by BlackDex** [reply]
Copy what? All is legal. There are several Bitwarden licensed features which we do not implement. Also the web-vault is build using the open source version.

**Comment by kpfleming** [comment]
If you have concerns about the legality of using Vaultwarden, you'll only be able to get those concerns addressed by consulting with your own legal counsel. Asking the Vaultwarden developers to tell you that it is 'legal' isn't sufficient.

Also, your definition of 'copy' appears to be 'implements the same network-level API, which is not a typical definition of 'copy'. A more appropriate word might be 'emulation', or 'compatible implementation'. All OSD-compliant open source licenses definitely do permit compatible implementations.
